src/Components/NavBar/Menus/NotificationsMenu.tsx

```diff
@@ -95,16 +95,19 @@ export const NotificationMenuItems: React.FC<
  * individual MenuItems for display. During fetch there is a loading spinner.
  */
 export const NotificationsMenu: React.FC = () => {
+  const isClient = typeof window !== "undefined"
```

Comment:
@alloy you mentioned an alternative (a local mounted state as a guard, and setting the state in `componentDidMount`).

Is that preferred due to not locking an implementation in to the web (`window` might be meaningless for React Native, for instance)?

If so, that's an easy update to make. However, these components are all functional so would have to become regular classes.

Reply:
By setting state we ensure that there's not an html mismatch between client and server. We have this `useDidMount` hook you can tap into: https://github.com/artsy/reaction/blob/master/src/Utils/Hooks/useDidMount.ts